### ISSUE TITLE
Wrongly escape quotes when running a unique test

### ISSUE THREAD

**Issue by Cobs** [Julien-R44/japa-vscode#4]
Hi,

I just got bamboozled for a couple of hours by a bug when trying to run a unique test in a file. Here an exemple :

```javascript
test("Can't do something", ({ assert }) => {
    assert.isTrue(false)
})
```

When clicking the "Run the below test" button, the terminal will execute the command `npm run test -- --files "tests/functional/myfile.spec.ts" --tests "Can\'t do something"`

as a result the terminal shows `NO TESTS EXECUTED`

Indeed, the right command should be `npm run test -- --files "tests/functional/myfile.spec.ts" --tests "Can't do something"`

**Comment by Julien-R44** [comment]
Oops !
Thanks for the feedback. I will fix it by tomorrow

**Comment by Julien-R44** [comment]
Published 1.0.0 on the marketplace!
